src/aks-preview/azext_aks_preview/_params.py

```diff
@@ -315,7 +315,7 @@ def load_arguments(self, _):
         c.argument('pod_cidrs')
         c.argument('service_cidrs')
         c.argument('load_balancer_managed_outbound_ipv6_count', type=int)
-        c.argument('enable_pod_security_policy', action='store_true', deprecate_info=c.deprecate(hide=True))
+        c.argument('enable_pod_security_policy', action='store_true', deprecate_info=c.deprecate(target='--enable-pod-security-policy', hide=True))
```

Comment:
From [doc](https://github.com/Azure/azure-cli/blob/dev/doc/authoring_command_modules/authoring_commands.md#deprecating-commands-and-arguments), it's not necessary?

Reply:
Without the target, the warning message is `Argument 'enable_pod_security_policy' has been deprecated and will be removed in a future release.`.

With the target, the warning message is `Argument '--enable-pod-security-policy' has been deprecated and will be removed in a future release.`. I think, this message is more clear.